pkg/deploy/plugin_registry.go

```diff
@@ -127,6 +131,8 @@ func SyncPluginRegistryToCluster(deployContext *DeployContext, cheHost string) (
 			}
 		}
 
+		deployContext.InternalService.PluginRegistryHost = fmt.Sprintf("%s://%s.%s.svc.cluster.local:%d/v3", PluginRegistryServiceProtocol, PluginRegistry, deployContext.CheCluster.Namespace, PluginRegistryServicePort)
```

Comment:
Same comment as for the devfile registry. This and the above defined  `routePathPrefix` (which is a URL and not a prefix) resolve to the same thing and therefore there's no need to have them both.

Reply:
Code was very changed after rebase with master.